fhir-replication-api/src/test/resources/config/default/fhir-server-config.json

```diff
@@ -3,9 +3,9 @@
         "persistence":{
             "replicationInterceptor": {
                 "enabled": true,
-                "studyIdExtensionUrl": "http://www.ibm.com/watson/health/fhir/extensions/whc-lsf/1.0/studyid",
-                "siteIdExtensionUrl": "http://www.ibm.com/watson/health/fhir/extensions/whc-lsf/1.0/siteid",
-                "subjectIdExtensionUrl": "http://www.ibm.com/watson/health/fhir/extensions/whc-lsf/1.0/patientid",
+                "studyIdExtensionUrl": "http://www.ibm.com/ibm/fhir/extensions/whc-lsf/1.0/studyid",
```

Comment:
ah, i missed the `http://www.ibm.com` variant...most of our extensions were just `http://ibm.com`

Reply:
can we remove the `www` while we're at it?

Reply:
please also remove the extra ibm.
so it should be `http://ibm.com/fhir/extensions/whc-lsf/1.0/studyid`